src/ifp/test/init_floorplan_polygon.tcl

```diff
@@ -0,0 +1,10 @@
+# init_floorplan called after a voltage domain is specified
```

Comment:
Wrong description.

Have you verified the routing tracks are correctly inside the polygon boundary?

Reply:
<img width="1224" alt="image" src="https://github.com/user-attachments/assets/1ae6105d-6a63-4108-8210-788a04f26007" />

They are not cut. Does this matter if there are routing obstructions? Or do we want to cut them similar to the standard cell rows?

Reply:
I suppose it won't matter as those areas are 'virtually obstructed'.